ddl/ddl_test.go

```diff
@@ -110,6 +110,7 @@ func TestT(t *testing.T) {
 	logutil.InitLogger(logutil.NewLogConfig(logLevel, "", "", logutil.EmptyFileLogConfig, false))
 	autoid.SetStep(5000)
 	ReorgWaitTimeout = 30 * time.Millisecond
+	batchInsertDeleteRangeSize = 2
```

Comment:
should add a benchmark test?

Reply:
If the benchmarks use mockTiKV, I don't think it makes much sense.

Reply:
@AilinKid PTAL